scss/_patterns_navigation.scss

```diff
@@ -21,67 +7,67 @@
     position: relative;
     width: 100%;
 
-    &__toggle--open {
-      @extend %navigation-toggle;
+    @media (min-width: $breakpoint-medium) {
+      display: flex;
+    }
+
+    &__toggle--open,
+    &__toggle--close,
+    &__link,
+    &__link>a {
+      &:visited {
+        color: $color-light;
+      }
+    }
+
+    &__link,
+    &__link>a {
+      &:hover {
+        text-decoration: underline;
+      }
     }
 
     &:target &__toggle--open {
       display: none;
     }
 
     &__toggle--close {
-      @extend %navigation-toggle;
       display: none;
     }
 
-    &:target &__toggle--close {
-      display: inline-block;
-    }
-
-    &__logo {
-      padding: 1rem;
-      text-align: center;
-      width: 100%;
-
-      @media (min-width: $breakpoint-medium) {
-        display: inline-block;
-        float: left;
-        width: auto;
-      }
+    &__toggle {
+      &--open,
```

Comment:
Nested selectors within media queries should have a carriage return preceding them, the older linter complained about this - it would seem the new one doesn't but if we could keep it consistent, that would be great!